gatsby_fp/src/components/Layout.js

```diff
@@ -35,9 +35,11 @@ export default function Layout({ children, data }) {
       {data
         ? <Seo
           title={`${sanitizedData.pf_first_name} ${sanitizedData.pf_last_name} - Stevens Institute of Technology`}
+          description={`Faculty Profile for ${sanitizedData.pf_first_name} ${sanitizedData.pf_last_name}`}
```

Comment:
No need to back this out since it's relatively minor, but in the future avoid using a single issue number for multiple unrelated things.